imports/plugins/core/revisions/server/publications.js

```diff
@@ -0,0 +1,24 @@
+import { Products, Revisions } from "/lib/collections";
```

Comment:
Probably none as I went with doing the revision publication from the products publication.

This file might need a rewrite or removal.